<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginSettingsUi</class>
 <widget class="QWidget" name="PluginSettingsUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Service</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="InformationWidget" name="m_InfoWidget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_StartButton">
       <property name="text">
        <string>Start Service</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_StopButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Stop Service</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="m_ApplyButton">
     <property name="text">
      <string>Apply Service Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PermissionWidget" name="m_PermissionWidget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_AgeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maxCount">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Content Rating</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_ContentRatingComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maxCount">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_GenderComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_LanguageComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_UrlDescriptionLabel">
     <property name="text">
      <string>Service URL or IP Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_UrlEdit">
     <property name="maxLength">
      <number>63</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_UserDisplayedNameLabel">
     <property name="text">
      <string>Title Of Service (Required)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_ServiceTitleEdit">
     <property name="maxLength">
      <number>127</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_DescriptionLabel">
     <property name="text">
      <string>Description Of Service (Optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="m_DescriptionEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ThumbnailChooseWidget" name="m_ThumbnailChooseWidget" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Greeting When Joined (Optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="m_GreetingEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Message If Rejected (Optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="m_RejectEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PermissionWidget</class>
   <extends>QWidget</extends>
   <header>PermissionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InformationWidget</class>
   <extends>QWidget</extends>
   <header>InformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ThumbnailChooseWidget</class>
   <extends>QWidget</extends>
   <header>ThumbnailChooseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
